collective-project-backend/Controllers/CompanyController.cs

```diff
@@ -27,7 +29,15 @@ public ActionResult<IList<Company>> GetAllCompanies()
             return Ok(_companyService.GetAllCompanies());
         }
 
-        [HttpPost]
+		[HttpGet]
+		[Route("companies/subscriptions")]
+		public ActionResult<IList<Company>> GetAllCompaniesWithSubscription() {
```

Comment:
please do not return objects from db if you don't need all the fields in frontend (for example a company has a user and frontend should not have access to it). Use view models instead.